docs/current/guides/snippets/openshift-gitlab/runner.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apps.gitlab.com/v1beta2
+kind: Runner
+metadata:
+  name: dagger-runner
+  namespace: dagger
+spec:
+  config: dagger-custom-config-toml
+  gitlabUrl: YOUR-GITLAB-URL
+  tags: dagger
+  token: YOUR-GITLAB-RUNNER-TOKEN-REFERENCE
```

Comment:
Can we explain where to look for this placeholder value, or how it is generated?

Reply:
Add some references to the official docs